api/client/service/create.go

```diff
@@ -27,7 +27,15 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.StringVar(&opts.mode, flagMode, "replicated", "Service mode (replicated or global)")
 	addServiceFlags(cmd, opts)
-	cmd.Flags().SetInterspersed(false)
+
+	flags.VarP(&opts.labels, flagLabel, "l", "Service labels")
+	flags.VarP(&opts.env, flagEnv, "e", "Set environment variables")
+	flags.VarP(&opts.mounts, flagMount, "m", "Attach a mount to the service")
```

Comment:
@dnephin what is this `m`, is it a `-m` short flag ? If so, can we please not have it ?